README.md

```diff
@@ -117,20 +117,21 @@ let make = () => {
 Using `fetchPolicy` to change interactions with the `apollo` cache, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#optionsfetchpolicy).
 
 ```reason
-let (_simple, full) = UserQuery.use(~fetchPolicy=NetworkOnly, ());
+let (_simple, full) = useQuery(~fetchPolicy=NetworkOnly, (module UserQuery));
 ```
 
 Using `errorPolicy` to change how errors are handled, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#optionserrorpolicy).
 
 ```reason
-let (simple, _full) = UserQuery.use(~errorPolicy=All, ());
+let (simple, _full) = useQuery(~errorPolicy=All, (module UserQuery));
 ```
 
 Using `skip` to skip query entirely, see [apollo docs](https://www.apollographql.com/docs/react/api/react-apollo/#configskip).
 
 ```reason
 let (simple, _full) =
-  UserQuery.use(
+  useQuery(
+    UserQuery.make(),
```

Comment:
I missed it when i was reviewing previous time. UseQuery should be pass as module as last parameter

Reply:
good catch!

Reply:
also use of reserved keyword `val`